README.md

```diff
@@ -210,7 +214,7 @@ The folder is structured in the following way: Any nonspecific-pangenome file is
       - 'test_2.fastq.gz'
     - sequencing_summary
       - 'test.sequencing_summary.txt'
-      - 'test2.sequencing_summary.txt' : A tab-delimited text file containing useful information for each read analysed during basecalling of nanopore sequencing data. 
```

Comment:
is it ok, that my formattor removed spaces at the end of those other lines or should I also undo that?

Reply:
That seems fine, I know that in the modules repo one of the linting checks complains if you have trailing spaces at the end of lines.